Objects/longobject.c

```diff
@@ -3790,6 +3860,41 @@ l_divmod(PyLongObject *v, PyLongObject *w,
     return 0;
 }
 
+/* Compute
+ *     *pmod = mod(v)
```

Comment:
I don't know what `mod(v)` could mean. The function also takes a `w` argument, which isn't mentioned here at all. Did this, e.g., mean to say `*pmod = v % w`?

Reply:
Fixed.